src/Scrapers/Pipeline/EslintCanaries/verify-workflow-strict-shell.canary.sh

```
@@ -0,0 +1,52 @@
+# REAL nesting `defaults.run.shell`, not three independent regex
+# matches that a malformed file with stray `shell:` lines could spoof.
+# Falls back to exit 1 if the package can't load — keeps the canary
+# fail-loud on every other class of error.
+if node --input-type=module -e "
+  import yaml from 'yaml';
+  import { readFileSync } from 'node:fs';
+  const doc = yaml.parse(readFileSync('${FILE}', 'utf8'));
+  const shell = doc?.defaults?.run?.shell ?? '';
+  const tokens = String(shell);
+  // Require bash + the bundled strict-mode flag pattern that
+  // `bash -euo pipefail` produces. Matching the literal '-euo' is
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)